examples/common/contour-config.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: contour
+  namespace: heptio-contour
+data:
+  contour.yaml: |
+    # should contour expect to be running inside a k8s cluster
+    # incluster: true
+    #
+    # path to kubeconfig (if not running inside a k8s cluster
+    # kubeconfig: /path/to/.kube/config
```

Comment:
Should this have the default mentioned? ie "~/.kube/config"?

Reply:
if we do that then someone might think that'll work. ~ is a shell expansion, if you call `os.Open("~/.kube/config")` it won't do what you expect. I think its better to have something obviously broken.